src/nni_manager/training_service/local/localTrainingService.ts

```diff
@@ -184,6 +184,18 @@ class LocalTrainingService implements TrainingService {
         return trialJob;
     }
 
+    public async getTrialLog(trialJobId: string, logType: LogType): Promise<string> {
```

Comment:
better to add unittest for this function

Reply:
Sure. On the way learning how to add nni_manager's ut...

Reply:
Added.